mk/PcapPlusPlus.mk.macosx.arm64

```diff
@@ -0,0 +1,11 @@
+### MAC OS X ###
+
+# includes
+PCAPPP_INCLUDES += -I$(MACOS_SDK_HOME)/usr/include/netinet
+
+# libs
+PCAPPP_LIBS += -lpcap -lpthread -framework SystemConfiguration -framework CoreFoundation
+
+# Apple Silicon (arm64) specific build flags
+CXXFLAGS += -arch arm64 -target arm64-apple-macos11 -isysroot $(MACOS_SDK_HOME)
+PCAPPP_BUILD_FLAGS += $(CXXFLAGS)
```

Comment:
I'd suggest the following:
- Keep only this part in `PcapPlusPlus.mk.macosx.arm64`
- In `configure-mac_os_x.sh`  you can do something like this:
  ```shell
  cat mk/PcapPlusPlus.mk.macosx >> $PCAPPLUSPLUS_MK
  if [ -n "$BUILD_FOR_ARM64" ]; then
    cat mk/PcapPlusPlus.mk.macosx.arm64 >> $PCAPPLUSPLUS_MK
  fi
  ```